index.html

```diff
@@ -345,123 +253,121 @@ <h5 class="social-media-p">LET'S CONNECT</h5>
               </svg>
             </span>
           </button>
-
           <a id="about-myself-skill-list-skills">Skills</a>
-
         </div>
 
-
       </div>
 
+    </section>
 
+    <!-- contact form -->
+    <section id="contact-form-base"></section>
+    <section id="contact-form-shapes">
+      <svg width="auto" height="auto" viewBox="0 0 222 710" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <rect x="157" y="29.3994" width="74.7012" height="17.2106" transform="rotate(-23.1765 157 29.3994)" fill="#7F8CFF"/>
+        <path d="M259.207 50.4103C255.738 44.3223 250.869 39.1276 244.993 35.2466C239.117 31.3657 232.4 28.9077 225.384 28.0716C218.369 27.2356 211.253 28.045 204.613 30.4343C197.973 32.8235 191.995 36.7256 187.163 41.8244C182.332 46.9233 178.782 53.0756 176.803 59.7833C174.823 66.491 174.468 73.5655 175.767 80.4339C177.066 87.3023 179.983 93.7717 184.281 99.3182C188.579 104.865 194.137 109.332 200.505 112.359L208.894 95.1011C205.266 93.3766 202.099 90.8314 199.651 87.6716C197.202 84.5117 195.541 80.8261 194.801 76.9132C194.06 73.0003 194.263 68.9699 195.39 65.1486C196.518 61.3272 198.54 57.8222 201.293 54.9174C204.046 52.0126 207.451 49.7896 211.234 48.4285C215.017 47.0673 219.071 46.6062 223.067 47.0825C227.064 47.5588 230.891 48.9591 234.239 51.1701C237.586 53.381 240.36 56.3405 242.336 59.8088L259.207 50.4103Z" fill="white"/>
+        <circle cx="10.5" cy="44.5" r="10.5" fill="#7F8CFF"/>
+        <rect width="76.175" height="13.6273" transform="matrix(0.801365 0.598176 0.598176 -0.801365 105.805 653.889)" fill="#7F8CFF"/>
+        <circle cx="122.228" cy="687.219" r="15.5552" transform="rotate(48.9435 122.228 687.219)" fill="white"/>
+      </svg>
     </section>
+    <section id="contact-form-data">
+      <h1 id="contact-form-data-h1">Contact me</h1>
+      <h6 id="contact-form-data-h6">If you have an application you<br> are interested in developing, a<br> feature that you need built or a<br> project that needs coding. I’d<br> love to help with it </h6>
+
+      <div id="shape"></div>
+
+      <form id="survey-form" action="https://formspree.io/f/{form_id}" method="post">
+        <input id="survey-form-name-input" name="Name" type="text" required="true" required placeholder="name" maxlength="30">
+        <input id="survey-form-email-input" name="Email" type="email" required="true" required placeholder="email">
+        <textarea id="survey-form-message-input" name="Message" type="text" required="true" required placeholder="Write your message here" maxlength="500"></textarea>
+        <button id="contact-form-data-button" type="submit">Get It Touch</button>
+      </form>
```

Comment:
- [ ] Your form looks great but it is not linked to a Formspree id so the user cannot send any message to you through this form. Please create a Formspree id and link to this form so it can be useful for you. You can follow [this link](https://formspree.io/html/) to get information about how to implement it.